### DISCUSSION TITLE
on ScaledJob can minReplicaCount be variable depending on time of day

### DISCUSSION THREAD

**Discussion by Eldarrin** [kedacore/keda#4878] [General]
Basically, the cluster is in heavy usage 16 hours a day and very light usage 8 hours a day. Can minReplicaCount be 3 for 16 hours and 0 for 8 hours.

I couldn't find a way to  make this work inside Keda.

Any thoughts or does it need writing?

Cheers, Andy

**Comment by JorTurFer** [comment]
You could set `minReplicaCount:3` and a cron trigger during the hours that you want with desired replicas 16. As the HPA controller will do a MAX between all the triggers, during the desired time window you will have at least the cron replica count.
Virtually you can do your goal. WDYT?

**Reply by Eldarrin** [reply]
max Desired is _n_ and won't change with time of day, only `minReplicaCount` and it's a job so does it still use the HPA?

**Reply by JorTurFer** [reply]
No, ScaledJob doesn't use HPA but you could use same max approach using `multipleScalersCalculation`.
About being the same, it's true but you can have multiple cron triggers to change it during different periods, as KEDA can do a MAX between them, you will have AT LEAST (as minimum) your cron desired replicas on each time. 

This is a huge hack to address the behavior taking into account that it's not supported natively, but it could work
